### ISSUE TITLE
Add option to group rows together

### ISSUE THREAD

**Issue by chrismytton** [everypolitician/compare_with_wikidata#52]
Currently the added/removed/modified rows are all jumbled together in the output. Adding an option to group rows of the same type together would potentially make some kinds of customization easier.

I believe [Listeria](https://www.wikidata.org/wiki/Template:Wikidata_list) already has an option for something similar to this, which I _think_ is the `section` option, is that right @tmtmtmtm?

**Comment by tmtmtmtm** [comment]
> I believe Listeria already has an option for something similar to this, which I think is the section option, is that right @tmtmtmtm?

Yes. It's not an exact equivalent, but it's a good example of the general concept.

**Comment by lucychambers** [comment]
Couple of hours timebox. If longer - let's rethink.
